docs/token-api.md

```diff
@@ -1,9 +1,9 @@
 # Token API
 
-The `Token API` offers a useful abstraction to deal with tokens in an implementation and blockchain independent way.
+The `Token API`, located in `token` and `token\token` packages, 
+offers a useful abstraction to deal with tokens in an implementation and backend independent way.
 
-Let us start with defining the tokens that the Token SDK handles.
-A token consists of the following triplet:
+The Token-SDK handles tokens that consist of the following triplet:
```

Comment:
Token SDK .... please check of consistent use of "Token-SDK" or "Token SDK". I guess it should be `Token SDK`